charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -29,6 +29,17 @@ data:
         common_name = {{ .common_name | quote }},
         {{- end }}
       }
+
+      {{- with .Values.federation }}
+      {{- if eq (.enabled | toString) "true" }}
+      federation {
+        bundle_endpoint {
+          address = "{{ .bundleEndpoint.address }}"
+          port = {{ .bundleEndpoint.port }}
+        }
```

Comment:
Ingress is just one possible way to expose this federation bundle endpoint. I'm not using it nor do I have a way to reliably test it.

I can try to add an ingress but I can't say when I will have time to do so nor can I guarantee that it works. Could we instead create an issue to add ingress for the bundle endpoint and someone can scratch that particular itch when they need it?

Reply:
@marcofranssen the federates with part is going to be more complicated. as it can also be a list...

He's also using istio so that can be exposed differently then ingress. I think the pr is as small as possible with the minimum implementation for some use cases. But needs further ones to really flesh out all of Federation. Should we merge the bit here and do follow-ons?